core/worker.go

```diff
@@ -614,6 +653,106 @@ func (w *worker) GeneratePendingHeader(block *types.WorkObject, fill bool) (*typ
 	return newWo, nil
 }
 
+func (w *worker) OrderTransactionSet(txs []*types.Transaction, gasUsedAfterTransaction []uint64, block *types.WorkObject) {
```

Comment:
First check if len(txs) == len(gasUsedAfterTransaction) for sanity

Reply:
Check done